scheduler/policy.cfg

```diff
@@ -0,0 +1,8 @@
+{
+"kind" : "Policy",
+"apiVersion" : "v1",
+"predicates" : [
+],
+"priorities" : [ 
+]
+}
```

Comment:
Since it is json, better align well and use .json

Reply:
The scheduler configuration file is a JSON file, which must be named policy.cfg, that specifies the predicates and priorities the scheduler will consider.

Above is the reason for having it that way and not .json

Reply:
No, seems this if force-code , can't use .json.

Reply:
https://docs.openshift.com/container-platform/4.3/nodes/scheduling/nodes-scheduler-default.html#nodes-scheduler-default-creating_nodes-scheduler-default

Reply:
Oh not so, oc create configmap -n openshift-config --from-file=**policy.cfg**=/path/to/file

Reply:
okay, if that is the case, then i think i will just send an update to this